src/main/java/uk/gov/ons/ctp/response/action/service/impl/ActionProcessingServiceImpl.java

```diff
@@ -261,6 +272,52 @@ private ActionRequest createActionRequest(final Action action, final ActionPlan
     return actionRequest;
   }
 
+  /**
+   * enrolment status for the case based off the enrolled parties
+   * @param childParty
+   * @return enrolment status
+   */
+  private String getEnrolmentStatus(final PartyDTO childParty) {
+    List<String> enrolmentStatuses = new ArrayList<>();
+    for (Association association : childParty.getAssociations()) {
+      for (Enrolment enrolment : association.getEnrolments()) {
+        enrolmentStatuses.add(enrolment.getEnrolmentStatus());
+      }
+    }
+    if (enrolmentStatuses.contains(ENABLED)) {
```

Comment:
What determines the priority of enabled vs pending etc. If the enrolmentStatuses contains all three why is enabled chosen first? Maybe a test around this would be handy

Reply:
You could have multiple respondents which is why there is an order of precedence. I will add a test thought.